capabilities_matrix/_topics/cloud_providers.md

```diff
@@ -0,0 +1,65 @@
+### Cloud Providers
+
+The following table outlines the capabilities of {{ site.data.product.title_short }} that are known to work for various cloud providers.
+
+| Feature                                    | Amazon Web Services (AWS) | Microsoft Azure | Google Cloud Platform (GCP) | Red Hat OpenStack Platform | IBM PowerVC via the OpenStack API     |
+| ------------------------------------------ | ------------------------- | --------------- | --------------------------- | -------------------------- | ------------------------------------- |
+| Relationship Discovery                     | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   |
+| Instance Drift Comparison                  | Yes                       | Yes             | No                          | Yes                        | No                                    |
+| Track Instance Genealogy                   | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   |
+| Capacity & Utilization                     | Yes                       | Yes             | Yes                         | Yes                        | No                                    |
+| Capture Cloud Layer Timelines              | No                        | No              | No                          | Yes                        | Yes                                   |
+| Capture Instance Event Timelines           | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   |
+| Discovery - Provider                       | Yes                       | Yes             | No                          | No                         | No                                    |
```

Comment:
Pretty sure we dropped discovery for cloud providers, let me confirm

Reply:
Yeah, no provider discovery for any cloud providers

Reply:
ok - so No across the board